Jenkinsfile

```diff
@@ -15,36 +15,6 @@ pipeline {
     timeout(time: 1, unit: 'HOURS') 
   }
   stages {
-    stage( 'docker.github.io' ) {
-      when {
-        expression { env.GIT_URL == 'https://github.com/Docker/docker.github.io.git' }
-      }
-      stages {
-        stage( 'build + push prod image, update prod swarm' ) {
-          when {
-            branch 'published'
-          }
-          steps {
-            withDockerRegistry(reg) {
-              sh """
-                docker build -t docs/docker.github.io:prod-${env.BUILD_NUMBER} .
-                docker tag docs/docker.github.io:prod-${env.BUILD_NUMBER} docs/docker.github.io:latest
-                docker push docs/docker.github.io:prod-${env.BUILD_NUMBER}
-                docker push docs/docker.github.io:latest
-                unzip -o $UCP_BUNDLE
-                cd ucp-bundle-success_bot
-                export DOCKER_TLS_VERIFY=1
-                export COMPOSE_TLS_VERSION=TLSv1_2
-                export DOCKER_CERT_PATH=${WORKSPACE}/ucp-bundle-success_bot
-                export DOCKER_HOST=$DOCKER_HOST_STRING
-                docker service update --detach=false --force --image docs/docker.github.io:prod-${env.BUILD_NUMBER} docs-docker-com_docs --with-registry-auth
-                curl -X POST -H 'Content-type: application/json' --data '{"text":"Successfully published docs. https://docs.docker.com/"}' $SLACK
-              """
-            }
-          }
-        }
-      }
-    }
     stage( 'docs-private' ) {
```

Comment:
The remaining Jenkinsfile after this PR only contains stages for `docs-private`. 
Should I keep it or just completely remove the Jenkinsfile in this PR?